tastypie_swagger/urls.py

```diff
@@ -1,13 +1,29 @@
+from django import __version__ as django_version
 try:
 	from django.conf.urls import patterns, include, url
 except ImportError:
 	from django.conf.urls.defaults import patterns, include, url
```

Comment:
Not working in Django 1.10: first import `ImportError: cannot import name patterns`, second `ImportError: No module named defaults`.